### PR TITLE
fix: Add children key in HeaderButtonsProps (#107)

### PR DESCRIPTION
**Pull request by alexakasanjeev**
#107

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -67,6 +67,7 @@ export interface HeaderButtonsProps {
    * However, you're free to use your own component (see `HeaderButton` for reference).
    */
   HeaderButtonComponent?: ComponentType<any>;
+  children: ReactChild | Array<ReactNode>;
```
Comment (vonovak) [inline index.d.ts:70]: seems to me that just `ReactNode` should be enough, no? ReactNode already includes array of children from what I can tell

**Review by vonovak — COMMENTED** [review]
Thanks for the PR! can you just answer one question I have? thanks!

**Comment by alexakasanjeev** [comment]
@vonovak yes, would be happy to answer any question, please ask

File: index.d.ts (hunk 1 above)
Comment (alexakasanjeev) [reply index.d.ts:70]: yes, I tested with `ReactNode` it is working fine, if you want I can push the new commit

**Comment by vonovak** [comment]
thanks for the PR! (the question was asked in the review comment). Thanks again.

**Pull request merged**